frontend/public/components/nav/items.tsx

```diff
@@ -307,7 +307,11 @@ export const PluginNavItems: React.FC<PluginNavItemsProps> = ({ items }) => {
         if (isSeparator(item)) {
           return <NavItemSeparator key={`separator-${index}`} />;
         }
-        return <li key={item.uid}>{createLink(item, true)}</li>;
+        return (
+          <div key={item.uid} className="co-nav-item">
```

Comment:
It looks like `co-nav-item` class name is not used anywhere?

Reply:
Yep should have been removed.